### ISSUE TITLE
Request: Export diff as its own text file

### ISSUE THREAD

**Issue by eccodev** [pnedev/comparePlus#400]
Title is self-explanatory. I'd like a way to export the compared differences between files as its own text file, a la MS Word.

**Comment by pnedev** [comment]
@eccodev ,
Do you mean generating a patch with all diffs like described in this issue: https://github.com/pnedev/comparePlus/issues/188 ?

**Comment by eccodev** [comment]
> @eccodev , Do you mean generating a patch with all diffs like the one described in this issue: #188 ?

Precisely, yeah! I wasn't sure how to properly word it, but that's exactly what I'm looking for.

**Comment by flywire** [comment]
Please close this duplicate Enhancement Request.

**Comment by pnedev** [comment]
Closing this request as it duplicates https://github.com/pnedev/comparePlus/issues/188
